doc/build/dialects/postgresql.rst

```diff
@@ -154,6 +154,68 @@ For example:
       during=DateTimeRange(datetime(2013, 3, 23), None)
   )
 
+MultiRange Types
+~~~~~~~~~~~
+
+The new MultiRange column types found in PostgreSQL 14 onwards are
+catered for by the following types:
+
+.. autoclass:: INT4MULTIRANGE
+
+
+.. autoclass:: INT8MULTIRANGE
+
+
+.. autoclass:: NUMMULTIRANGE
+
+
+.. autoclass:: DATEMULTIRANGE
+
+
+.. autoclass:: TSMULTIRANGE
+
+
+.. autoclass:: TSTZMULTIRANGE
+
+
+The types above get most of their functionality from the following
+mixin:
+
+.. autoclass:: sqlalchemy.dialects.postgresql.ranges.RangeOperators
+    :members:
+
+.. warning::
+
+  The multirange type DDL is currently only supported by the
+  psycopg (a.k.a psycopg3) PostgreSQL DBAPI driver.
```

Comment:
**mike bayer** (zzzeek) wrote:

really? the DDL or the whole thing?  why is this?  this seems very strange, typically the only place a DBAPI is involved is in passing the data to/from the column, which isn't DDL, and in that case we will format the data in a way that the driver can send/receive as a string.     is that not possible here?

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3728

Reply:
Yeah I could use some help here... should I stick with the working the wording from the PostgreSQL Range type [docs](https://docs.sqlalchemy.org/en/14/dialects/postgresql.html?highlight=range#range-types)
, i.e.
```
.. warning::
  The multirange type DDL support should work with any PostgreSQL DBAPI
  driver, however the data types returned may vary.
```

Reply:
OK, that's disastrous that this is our implementation.   this is from many many years ago when we basically had just one driver.    it looks like the whole thing is based on hardcoding to that datatype that's in "extras".  this all has to be fixed at some point.

so yes I would use the same docstring that it should work with any driver but we haven't smoothed out the return types.     fixing this very bad situation would address all the range/multirange types at once.